Source/WebKit/WebProcess/GPU/media/RemoteAudioDestinationProxy.h

```diff
@@ -65,7 +65,7 @@ class RemoteAudioDestinationProxy final : public WebCore::AudioDestinationResamp
 
     void startRenderingThread();
     void stopRenderingThread();
-    void renderQuantum();
+    void renderQuantum(unsigned frameCount);
```

Comment:
Let's rename to renderAudio or something like that, we are no longer really rendering a quantum

Reply:
Ok.